### PR TITLE
Automated Napoleonic wars narrative input

### LINKED ISSUE #2: Input first narrative Napoleonic wars
https://docs.google.com/spreadsheets/d/1_2JcshfUBgZj7Ec5x31RjiTwdSMCtPpTZXOthIsK39I/edit?usp=sharing
Text, dates, battles and peace treaties.
Author http://www.historyworld.net

### PR DESCRIPTION
**Pull request by quorth0n**
Fixes #2. I wasn't expecting it to be this much code and as such it's not pretty. I'm aware we're pressing for a deadline soon and to my knowledge this code shouldn't be necessary after the narrative input interface is complete so I pushed it for now. If it's deemed necessary however I'm willing to refactor into a cleaner OOP approach.

### CODE REVIEW

**Comment by quorth0n** [comment]
Note that much of this code is dependant on https://github.com/chronhq/backend/pull/42

**Review by MiklerGM — CHANGES_REQUESTED** [review]
Images are missing, they are located in pics column in gsheet

File: narratives/napoleon/input.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+)
+
+with open("data.csv", mode="r") as narrative_file:
+    narrative_reader = csv.DictReader(narrative_file)
+    for row in narrative_reader:
+        # TODO: fix placeholder MapSettings
+        ms_data = {
+            "bbox": "MULTIPOINT(-15.751 35.382, 51.046 62.222)",
+            "zoom_min": 1,
+            "zoom_max": 15,
```
Comment (MiklerGM) [inline narratives/napoleon/input.py:91]: I think that is too much.

**Review by MiklerGM — APPROVED** [review]
Looks good,
Map Settings and zoom parameters should be tweaked later. I think we will go with 3 to 8 for zoom, and more precise bboxes.

File: narratives/napoleon/input.py (hunk 1 above)
Comment (quorth0n) [reply narratives/napoleon/input.py:91]: Completely agree, like you said I expect these values to be tweaked later

**Pull request merged**